oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/oidc/authentication/OidcIdTokenValidator.java

```diff
@@ -132,6 +133,7 @@ public OAuth2TokenValidatorResult validate(Jwt idToken) {
 	 */
 	public final void setClockSkew(Duration clockSkew) {
 		Assert.notNull(clockSkew, "clockSkew cannot be null");
+		Assert.isTrue(clockSkew.compareTo(MINIMUM_CLOCK_SKEW) >=0, "clockSkew should be greater than 0");
```

Comment:
Let's remove `MINIMUM_CLOCK_SKEW` and update the assertion as follows:

`Assert.isTrue(clockSkew.getSeconds() >= 0, "clockSkew must be >= 0");`